packages-backend/loggers/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@babel/runtime": "^7.22.15",
     "@babel/runtime-corejs3": "^7.22.15",
-    "@sentry/node": "7.81.0",
+    "@sentry/node": "7.88.0",
```

Comment:
@emmenko I don't see this dependency imported anywhere.
Do you maybe remember why it is here?

Reply:
Ah good catch. It's a leftover from #1472 when we open sourced our logger package. Back then we had sentry tracking included for our backend services.

We can remove this dependency now. Thanks